Source/Editor/DualityEditor/PackageManagement/PackageManager.cs

```diff
@@ -145,24 +153,13 @@ public PackageManager(PackageManagerEnvironment workEnvironment, PackageSetup pa
 			this.setup = packageSetup;
 
 			// Create internal package management objects
-			IPackageRepository[] repositories = this.setup.RepositoryUrls
-				.Select(x => this.CreateRepository(x))
-				.Where(x => x != null)
-				.ToArray();
-			this.repository = new AggregateRepository(repositories);
-			this.manager = new NuGetTargetedPackageManager(
-				new FrameworkName(".NETFramework", new Version(4, 5, 2)), 
-				this.repository, 
-				this.env.RepositoryPath);
+			var nuGetFramework = NuGetFramework.Parse("net472");
+			var settings = NuGet.Configuration.Settings.LoadDefaultSettings(root: null);
+			this.logger = new PackageManagerLogger();
+			this.manager = new NuGetTargetedPackageManager(nuGetFramework, this.logger, settings, this.env.RepositoryPath);
```

Comment:
It seems like the package repository URLs are currently ignored entirely, which also breaks most of the package tests because they create a local repository to source from. How do we specify repository URLs with NuGet 5?

Reply:
I believe this is possible and I think I saw the API somewhere but currently not at my pc to check this. Will check it later.